File: Java/백준/Bronze/19602. Dog Treats/Dog Treats.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt()+in.nextInt()*2+in.nextInt()*3;
        System.out.println((a>=10)?"happy":"sad");
    }
}

File: C#/백준/Bronze/19602. Dog Treats/Dog Treats.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = 0;
            for (int i=1; i<4; i++){
                a+=int.Parse(Console.ReadLine())*i;
            }
            Console.WriteLine((a>=10)?"happy":"sad");
        }
    }
}